charts/scalardb-analytics-postgresql/README.md

```diff
@@ -0,0 +1,54 @@
+# scalardb-analytics-postgresql
```

Comment:
This file is automatically generated based on the `values.yaml` file.

Reply:
Just curious. How is this file generated?

Reply:
You can generate this file by using `helm-docs`.
https://github.com/norwoodj/helm-docs

Also, in our repository, you can run the `helm-docs` by using the following script.
https://github.com/scalar-labs/helm-charts/blob/main/scripts/update-chart-docs.sh